modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -0,0 +1,26 @@
+= Premium upgrade promotion
+:navtitle: {productname} Premium upgrade promotion
+:description: Editor options related to turning the Premium promotion display off
+:keywords: upgrade, promotion, premium, button
+
+== Premium upgrade promotion defaults
+
+When {productname} 6.2 or later is running in self-hosted Community mode, an *Upgrade* promotion button appears in the top-right of the editor menu bar by default. The button does note appear when {productname} is running in Community mode on the Tiny cloud.
+
+This promotion button redirects to a link:{companyurl}/tinymce-self-hosted-premium-features/[promotion page], showcasing the benefits of the variety of xref:plugins#premium-plugins[Premium Plugins] available for {productname}.
+
+When {productname} 6.2 or later is running in Premium mode, the *Upgrade* promotion button is disabled. And it is disabled when running on a Premium plan whether {productname} is running in the {cloudname} or as a self-hosted instance.
+
+To turn this button off in Community mode, set the `promotion` option, which defaults to `true`, to false.
```

Comment:
Doesn't having `which defaults to true` between `set` and `false` making this more confusing? Legitimate question.

Reply:
This has been re-written and the sub-clause is now a sentence in its own right.

Simple declarative sentences are, generally speaking, less likely to cause confusion. And hammering home the point through repetition is a hallmark of effective technical communications.

As Jazz bassist Adam Neely puts it, _repetition legitimises_.